examples/tanstack-router/client/src/utils/router.tsx

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import {
+  Link,
+  Outlet,
+  createReactRouter,
+  createRouteConfig,
+  useMatch,
+} from '@tanstack/react-router';
+import { getPostById, getPosts } from '../../../server/fetchers';
```

Comment:
Importing stuff from the server here shouldn't be needed, right? See comments further down